### PR TITLE
Issue #SB-15992 fix:Moving out the label changes to documentation team

### PR DESCRIPTION
**Pull request by ArjunKhetia-NIIT**

### CODE REVIEW

File: src/app/gulpfile.js [hunk 1]
```diff
@@ -14,6 +14,13 @@ const collectionEditor = process.env.sunbird_collection_editor_artifact_url;
 const genericEditor = process.env.sunbird_generic_editor_artifact_url;
 const editorsDestPath = 'client/src/thirdparty/editors/'
 
+// PhraseApp Configuration
+const authToken = 'Basic YXJqdW5raGVjeaE6QXJqdW5AMTk5MA==';
```
Comment (rajeshkumaravel) [inline src/app/gulpfile.js:18]: Configuration cannot be directly used here; it will be public.

CC: @rajeevsathish

File: src/app/gulpfile.js [hunk 2]
```diff
@@ -14,6 +14,13 @@ const collectionEditor = process.env.sunbird_collection_editor_artifact_url;
...
+const project = 'DIKSHA Portal,Sunbird Creation ';
```
Comment (rajeshkumaravel) [inline src/app/gulpfile.js:19]: Tenant references should not be there

Comment (ArjunKhetia-NIIT) [reply src/app/gulpfile.js:19]: There is No Tenant references It is Phraseapp Project Name.

File: src/app/gulpfile.js (hunk 1 above)
Comment (ArjunKhetia-NIIT) [reply src/app/gulpfile.js]: Done bro.

**Pull request closed without merging**